src/grid/submission/SubmissionGridHeader.component.ts

```diff
@@ -31,21 +32,21 @@ export class SubmissionGridHeaderComponent extends GridHeaderComponent {
   }
 
   // Set header for both component and column
-  setHeader(column?: any, component?: ExtendedComponentSchema, sort?: EventEmitter<any>) {
-    const key = column ? column.path : `data.${component.key}`;
+  setHeader(column?: GridColumn, component?: ExtendedComponentSchema, sort?: any) {
```

Comment:
@vlad-shusterman could you please provide type for `sort?` here? It should be `SortType` as I see